src/Umbraco.Core/Cache/DistributedCacheExtensions.cs

```diff
@@ -32,7 +32,13 @@ public static void RefreshUserCache(this DistributedCache dc, int userId)
         => dc.Refresh(UserCacheRefresher.UniqueId, userId);
 
     public static void RefreshUserCache(this DistributedCache dc, IEnumerable<IUser> users)
-        => dc.Refresh(UserCacheRefresher.UniqueId, users.Select(x => x.Id).Distinct().ToArray());
+    {
+        foreach (IUser user in users)
+        {
+            dc.Refresh(UserCacheRefresher.UniqueId, user.Key);
+            dc.Refresh(UserCacheRefresher.UniqueId, user.Key);
```

Comment:
Duplicate line here? Or am I not reading this right? 🙈

Reply:
Good catch, was supposed to be id & key 🙈